environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
Agreed, it should, and it very well already might :-) I'm just hoping that during the course of this PR we can re-verify that that's the case.